node/test/go/dev-mode/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "dev-mode-tests",
+  "version": "1.0.0",
+  "description": "Dev mode integration test dependencies",
+  "repository": "serverless/console",
+  "author": "Serverless, Inc.",
+  "license": "MIT",
+  "dependencies": {
+    "@serverless/sdk-schema": "^0.9.0"
```

Comment:
Let's remove this and related files

All _test_ dependencies are _dev_ dependencies, and for this repository we configure them in single place at `node/package.json`

Reply:
Ahh okay yeah that would definitely make things easier 😅

Reply:
Okay I moved the dependency 👍 I don't know why it didn't occur to me to do that before but this makes a lot more sense 😎